ReactAndroid/build.gradle

```diff
@@ -231,12 +231,11 @@ task packageReactNdkLibsForBuck(dependsOn: packageReactNdkLibs, type: Copy) {
 }
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
```

Comment:
Actually, we rely on `buildToolsVersion` in the CI. That's why `test_android` is failing.

https://github.com/facebook/react-native/blob/b63015d34ef51585c452142f666d4f36ceaa1b56/scripts/validate-android-test-env.sh#L55

Reply:
The new gradle version will choose the build tools version itself, but specifying it here won't (afaik) hurt either.